src/librustdoc/html/static/rustdoc.css

```diff
@@ -1757,14 +1764,19 @@ div.name.expand::before {
 	top: 2px;
 }
 
+details {
+	position: relative;
+}
```

Comment:
Elsewhere in this file, we have:

```
details.rustdoc-toggle[open] {
	position: relative;
}
```

Rather than add a new rule, we should modify the existing rule to apply to both open and closed details (i.e. remove the `[open]`).

Reply:
A bit differently, we need to have instead:

```
details.rustdoc-toggle, details.undocumented {
	position: relative;
}
```